content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -5,30 +5,40 @@ weight: 65
 keywords: [security,health-check]
 ---
 
-This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) for health checking of Istio services.
+This task shows how to use [Kubernetes liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+for health checking of Istio services.
 
 There are three options for liveness and readiness probes in Kubernetes:
 
 1. Command
 1. HTTP request
 1. TCP request
 
-This task provides examples for the first two options with Istio mutual TLS enabled and disabled, respectively.
+This task provides examples for the first two options with Istio mutual TLS
```

Comment:
The following task provides examples for doing liveness and readiness probes through the command line and through HTTP requests. The examples cover both situations: mutual TLS enabled and mutual TLS disabled.

Reply:
done